### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by nathan-roberts** [schillermann/trolleydienst-php#16] [Q&A] [answered]
Hi,

Thanks for all your hard work and for making this publicly available.

I'd like to use this for my local congregation, would you be happy if I added some simple translation and made a pull request?

Regards

Nathan

**Comment by schillermann** [comment, marked as answer]
Hi @nathan-roberts, 

translation is available with [version 1.10.0](https://github.com/schillermann/trolleydienst-php/releases/tag/1.10.0). 

Greetings
Mario

**Reply by nathan-roberts** [reply]
Thanks @schillermann I actually work with @mikecopestake glad to see you merged the PR

**Reply by schillermann** [reply]
OK, you know @mikecopestake?
